### ISSUE TITLE
[ BUG ] `Get-FalconHorizonIom` with `All` is not paginating properly

### ISSUE THREAD

**Issue by bk-cs** [CrowdStrike/psfalcon#295]
**Describe the bug**
When using `Get-FalconHorizonIom` with `All`, various errors are encountered, including:
```
Write-Result : [{"code":400,"message":"Invalid Request limit, maximum allowed is 1000"}]
```
```
Write-Result : [{"code":400,"message":"cannot provide both nextToken and cloud_provider parameters"}]
```
These same errors will likely apply to `Get-FalconHorizonIoa` due to their similarities.

**Environment (please complete the following information):**
 - OS: Windows 11
 - PowerShell: 5.1
 - PSFalcon: 2.2.4

**Comment by bk-cs** [comment]
This has been fixed for the 2.2.5 release. If you'd like to fix it in your local module before release, please update the following files with those from the 2.2.5 repo:

`Public\detects.ps1`
```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.5/public/detects.ps1).content > $HOME\Documents\WindowsPowerShell\Modules\2.2.4\Public\detects.ps1
```
`Private\Private.ps1`
```powershell
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.5/private/Private.ps1).content > $HOME\Documents\WindowsPowerShell\Modules\2.2.4\Private\Private.ps1
```

**Comment by bk-cs** [comment]
[2.2.5 release](https://github.com/CrowdStrike/psfalcon/releases/tag/2.2.5)
